TelCo.ColorCoder/TestColorCoder.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Drawing;
+using System.Diagnostics;
+using TelCo.ColorCoder;
+
+
+namespace TestColorCoder
+{
+    public class TestColorCoder
+    {
+        /// <summary>
+        /// Test code for the class
+        /// </summary>
+        /// <param name="args"></param>
+        private static void Main(string[] args)
+        {
+            // ColorCoder.printReferenceManual();
+            int pairNumber = 4;
+            ColorPair testPair1 = ColorCoder.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.White);
+            Debug.Assert(testPair1.minorColor == Color.Brown);
+
+            pairNumber = 5;
+            testPair1 = ColorCoder.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.White);
+            Debug.Assert(testPair1.minorColor == Color.SlateGray);
+
+            pairNumber = 23;
+            testPair1 = ColorCoder.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.Violet);
+            Debug.Assert(testPair1.minorColor == Color.Green);
+
+            ColorPair testPair2 = new ColorPair() { majorColor = Color.Yellow, minorColor = Color.Green };
+            pairNumber = ColorCoder.GetPairNumberFromColor(testPair2);
+            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n", testPair2, pairNumber);
+            Debug.Assert(pairNumber == 18);
+
+            testPair2 = new ColorPair() { majorColor = Color.Red, minorColor = Color.Blue };
+            pairNumber = ColorCoder.GetPairNumberFromColor(testPair2);
+            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}", testPair2, pairNumber);
+            Debug.Assert(pairNumber == 6);
```

Comment:
How about adding a test for `printReferenceManual`?

Reply:
I am not sure how would I go about doing this. `printReferenceManual` simply prints out a table of the pair number and corresponding color pair on the console, so I don't see a way to test this function. The output of `printReferenceManual` function is given below -
```
This is the reference manual for the 25-pair color code
======================================================

Code |  Major Color  |  Minor Color  
-----+---------------+---------------
 1   |  White        |  Blue         
 2   |  White        |  Orange       
 3   |  White        |  Green        
 4   |  White        |  Brown        
 5   |  White        |  SlateGray    
 6   |  Red          |  Blue         
 7   |  Red          |  Orange       
 8   |  Red          |  Green        
 9   |  Red          |  Brown        
 10  |  Red          |  SlateGray    
 11  |  Black        |  Blue         
 12  |  Black        |  Orange       
 13  |  Black        |  Green        
 14  |  Black        |  Brown        
 15  |  Black        |  SlateGray    
 16  |  Yellow       |  Blue         
 17  |  Yellow       |  Orange       
 18  |  Yellow       |  Green        
 19  |  Yellow       |  Brown        
 20  |  Yellow       |  SlateGray    
 21  |  Violet       |  Blue         
 22  |  Violet       |  Orange       
 23  |  Violet       |  Green        
 24  |  Violet       |  Brown        
 25  |  Violet       |  SlateGray  
 ```

Reply:
The thing I can do is simply call this function so that it prints on the console. Is that what you meant?